### DISCUSSION TITLE
Pull from WayBackMachine archive.org

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [ArchiveBox/ArchiveBox#1280] [Q&A] [answered]
Hello at all

Is it possible to pull missing informations from the WayBackMachine?
I tried to get an old article. Its already completly offline and still exists on the WBM.
Now I entered the Link from the WBM and archivebox only archives this one url. All other urls are linked to the originial website.

Is there a fallback or an option to download the missing pages directly from WBM?

**Comment by pirate** [comment, marked as answer]
Duplicate of #160, subscribe to that issue for progress.
